scripts/travis/before_install.sh

```diff
@@ -5,18 +5,31 @@ set -o pipefail
 
 if [[ ${PLATFORM} == "android" ]]; then
 
-    # Install android ndk
-    echo "Cloning mindk..."
-    git clone --quiet --depth 1 --branch deploy-all-arch https://github.com/tangrams/mindk.git
-    export ANDROID_NDK=$PWD/mindk/android-ndk-r10e
+    # Note: the right way to download these packages is through the Android Studio SDK manager,
+    # these steps should be removed when/if ndk-bundle and cmake become available from the
+    # command-line SDK update tool.
+
+    # Download android ndk
+    echo "Downloading ndk..."
+    curl -L https://dl.google.com/android/repository/android-ndk-r13b-linux-x86_64.zip -o ndk.zip
```

Comment:
We can I think, will require to update travis configuration to use their caching, but based on the email they send me, it seems they copy the contents to a s3 bucket and download it from their in the next travis run, which will take equally long. So no use as such!